frontend/lib/src/components/core/Block/Block.tsx

```diff
@@ -201,6 +201,7 @@ function ScrollToBottomVerticalBlockWrapper(
       border={border}
       height={height}
       data-testid="stVerticalBlockBorderWrapper"
+      data-test-scroll-behavior="scroll-to-bottom"
```

Comment:
O cool didn't know about this.

Reply:
I cannot reuse `data-testid` since I want to keep it the same regardless of which scrolling behavior is used (so that the container maintains the same hierarchy of test ids. So, needed to introduce an alternative attribute to check that in the tests.